benchmarks/index.js

```diff
@@ -0,0 +1,43 @@
+var DataSource = require('loopback-datasource-juggler').DataSource;
+var connector = require('..');
+var Benchmark = require('benchmark');
+
+var ds = new DataSource(connector, {
+  host: process.env.LB_HOST || '127.0.0.1',
+  port: process.env.LB_PORT || 27017,
+  database: process.env.LB_DB || 'strongloop'
+});
+var Todo = ds.define('Todo', {
+  content: {type: String}
+});
+
+var suite = new Benchmark.Suite;
+suite
+  .on('start', function() {
+    console.log('#', new Date());
+  })
+  .add('Todo.create', {
+    defer: true,
+    fn: function(deferred) {
+      Todo.create({content: 'Buy eggs'}, function() {
```

Comment:
Please use a unique value here, e.g. `'Buy eggs' + (tid++)` where `var tid = 0` is set somewhere outside of `fn`. While it's probably not critical for MongoDB, it can uncover inefficiencies in SQL connectors.